tasks/Qc.wdl

```diff
@@ -288,6 +288,7 @@ task BuildBamIndex {
     File ref_fasta
     File ref_fasta_index
     Int preemptible_tries
+    String bam_index_file_name = sub(input_bam,"am$","ai")
```

Comment:
If we're going this route, I think we need to pass it as `OUTPUT=` to the Picard tool